cas-server-core-util/src/main/java/org/jasig/cas/util/http/SimpleHttpClientFactoryBean.java

```diff
@@ -428,6 +429,14 @@ public void setRedirectsEnabled(final boolean redirectsEnabled) {
     }
 
     /**
+     * Destroy.
+     */
+    @PreDestroy
+    public void destroy() {
+        this.executorService.shutdownNow();
```

Comment:
Verified. I think the shutdown call above should take care of executor pool I believe.